### PR TITLE
Regroup byron commands

### PR DESCRIPTION
**Pull request by Jimbo4350**
Issue
-----------

- Group byron commands similarly to the shelley commands.

- This PR **results** in breaking changes to upstream dependencies.

Checklist
---------
- [x] I have added the appropriate labels to this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -353,13 +353,13 @@ See the [Byron specification](https://hydra.iohk.io/job/Cardano/cardano-ledger-s
...
 ## Update proposal voting
-You can create and submit byron update proposal votes with the `create-byron-proposal-vote` & `submit-byron-proposal-vote` commands. The following are two example commands:
+You can create and submit byron update proposal votes with the `create-proposal-vote` & `submit-proposal-vote` commands. The following are two example commands:
 
 
 Byron vote creation:
 
 ```
-cabal exec cardano-cli -- byron create-byron-proposal-vote
+cabal exec cardano-cli -- byron create-proposal-vote
```
Comment (ArturWieczorek) [inline README.md:362]: This needs to be changed to `cabal exec cardano-cli -- byron node create-proposal-vote` there is a missing `node` part, the same goes for located above commands for creating and submitting update proposal:
```
cabal exec cardano-cli -- byron node create-update-proposal
cabal exec cardano-cli -- byron node submit-update-proposal
```

File: README.md [hunk 2]
```diff
@@ -370,7 +370,7 @@ cabal exec cardano-cli -- byron create-byron-proposal-vote
 Byron vote submission:
 
 ```
-cabal exec cardano-cli -- byron submit-byron-proposal-vote
+cabal exec cardano-cli -- byron submit-proposal-vote
```
Comment (ArturWieczorek) [inline README.md:373]: As above, a `node` part needs to be added: `cabal exec cardano-cli -- byron node submit-proposal-vote`

**Comment by ArturWieczorek** [comment]
LGTM,works, **however** README needs to be updated so all those commands have `node` part in them as it is currently missing:

```
cabal exec cardano-cli -- byron node create-update-proposal
cabal exec cardano-cli -- byron node submit-update-proposal
cabal exec cardano-cli -- byron node create-proposal-vote
cabal exec cardano-cli -- byron node submit-proposal-vote
```

**Comment by Jimbo4350** [comment]
bors merge

**Pull request merged**